graphql-dgs-codegen-core/src/main/kotlin/com/netflix/graphql/dgs/codegen/generators/java/ClientApiGenerator.kt

```diff
@@ -188,7 +188,7 @@ class ClientApiGenerator(private val config: CodeGenConfig, private val document
                 .build()
         )
         javaType.addType(builderClass.build())
-        return JavaFile.builder(getPackageName(), javaType.build()).build()
+        return JavaFile.builder(getPackageName().plus("." + operation.lowercase()), javaType.build()).build()
```

Comment:
I think this will break current customers but I do like the approach.
@srinivasankavitha what do you think?

Reply:
Yes, this will be a breaking change. Might I suggest disambiguating by having GraphQLQuery in general, and then GraphQLMutation/GraphQLSubscription in cases where there are duplicates to be handled?

Reply:
Hi @srinivasankavitha, yes, there will be a breaking change and at first I thought about it but didn't try more. Thank you for your advice, I will try to modify it by following your advice.
Thanks!

Reply:
Hi @srinivasankavitha , @berngp , tried another solution.
Please help to check this is available？
Thanks very much!